### PR TITLE
Terraform local dev setup

### PR DESCRIPTION
**Pull request by m-abboud**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,4 +110,25 @@ The production site can be found at [docs.vellum.ai](https://docs.vellum.ai/).
...
-to the Vellum org in Fern, which requires contacting the fern team in #fern-x-vellum in Slack.
\ No newline at end of file
+to the Vellum org in Fern, which requires contacting the fern team in #fern-x-vellum in Slack.
+
+
+## Terraform CDKTF Local Testing
+To test changes to the Vellum terraform provider you need to create a file at `~/dev.tfrc` with these contents:
+
+```
+provider_installation {
```
Comment (dvargas92495) [inline README.md:120]: Woah nice! Looks like this was the right track. Putting this in the default `.terraformrc` would've also worked right? I guess in that world we would've needed a different way to alias dev vs prod

File: package.json [hunk 2]
```diff
@@ -1,6 +1,7 @@
 {
   "scripts": {
     "cdktf": "ts-node ./scripts/cdktf.ts",
+    "cdktf:local": "TF_CLI_CONFIG_FILE=~/dev.tfrc ts-node ./scripts/cdktf.ts",
```
Comment (dvargas92495) [inline package.json:4]: [nit] Would be a fan of:
- adding a `dev.tfrc.example`
- adding `dev.tfrc` to the git ignore for now

**Review by dvargas92495 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (m-abboud) [reply README.md:120]: Yes but then you wouldn't be able to grab the prod stuff so I did it like this

File: package.json (hunk 2 above)
Comment (m-abboud) [reply package.json:4]: > adding dev.tfrc to the git ignore for now
It won't load from project root if you just put the file name so I thought just having it in home dir would spare people from having to put in an extra path

Comment (m-abboud) [reply package.json:4]: oh ya right it's also convenient in the home dir since you need to pass this param when doing `cdktf deploy` when testing it

**Pull request merged**